frontend/src/Frontend/UI/Wallet.hs

```diff
@@ -127,16 +127,16 @@ uiKeyItems model = do
     keyMap' = model ^. wallet_accounts
     keyMap = Map.fromAscList . IntMap.toAscList <$> keyMap'
     tableAttrs =
-      "style" =: "table-layout: fixed; width: 100%"
+      "style" =: "table-layout: fixed; width: 98%"
```

Comment:
How about leaving it at 100% and use a fixed margin?

Reply:
Something about it being at 100% was causing a scrollbar. And it's too late to figure out exactly why. It's more of a workaround to get rid of the scrollbar than just a margin.